docs/sdk/database/aliases.md

```diff
@@ -0,0 +1,114 @@
+
+Under the hood, `aliases` contains an object with `read` and `write` functions. The `read` function must return a mapping of the project-scoped table alias to the universally unique Tableland table nam—for example, `{ "users": "users_31337_2" }`. The `write` function will be called any time a `Database` instance is used to create a table and modifies the JSON file, writing the mapping to it.
+
+:::note
+All project aliases must be unique. If you try to create a table with a preexisting alias saved in your config file, you'll run into an error: `table name already exists in aliases`.
+:::
+
+## Configuration file
+
+As a best practice, the alias config file should exist in your project's directory to ensure the mappings from table aliases to the generated names are persisted and do not get lost. First, create a JSON file with an empty object (`{}`) in the root of your project (or a subdirectory). It will ultimately contain all of your table aliases. Let's name the file `tableland.config.json`, but it can be any file name you'd like.
+
+```sh
+echo '{}' > tableland.config.json
+```
+
+Then, import `jsonFileAliases` from the SDK's `helpers` into your source code; this is a utility function that allows you to simply provide the path to your aliases file.
+
+```js
+import { Database } from "@tableland/sdk";
+import { jsonFileAliases } from "@tableland/sdk/helpers";
+
+const db = new Database({
+  signer,
+  // highlight-start
+  // Specify the path to your aliases config file
+  aliases: jsonFileAliases("./tableland.config.json"),
+  // highlight-end
+});
+```
+
+You'll start by creating a table. Note how the `prepare` statements for writes and reads make use of the table prefix used in the create statement instead of the raw, generated table name (`{prefix}_{chainId}_{tableId}`).
+
+```js
+// Create a table with the custom prefix `main`, which is then used as an alias
+const { meta: create } = await db
+  .prepare(`CREATE TABLE main (id integer primary key, val text);`)
+  .run();
+await create.txn?.wait();
+
+// Write to the table using the alias `main`, instead of the raw table name
+const { meta: write } = await db
+  .prepare(`INSERT INTO main (val) VALUES ('test')`)
+  .run();
+await write.txn?.wait();
+
+// Also, read from the table with the alias
+const { results } = await db.prepare(`SELECT * FROM main`).all();
+```
+
+Creating a table will automatically write the mapping to the `tableland.config.json` file. The config file will look something like the following (e.g., if you create a `main` prefixed table using Local Tableland):
+
+```json
+{
+  "main": "main_31337_2"
+}
+```
+
+## Multiple table aliases
+
+One important callout: every time you create a table, it will overwrite mappings in the file specified in the `aliases` parameter. So, if you were to execute `CREATE TABLE` statements individually instead of batching them together, the config file will be overwritten upon each statement successfully resolving.
```

Comment:
Reading this makes me realize we might want to change the helper to add to the aliases, instead of overwriting.  Does that make more sense?
We can leave it for this PR, but maybe open an issue to change this.

Reply:
good catches above, changed! 

and yeah, i had the same feeling. i'd think i'd rather have the full history of table names because overwriting can make you lose them permanently. i'll open an issue.